include/triton/Tools/Sys/GetEnv.hpp

```diff
@@ -55,6 +56,18 @@ inline std::string getenv(const char *name) {
   return result;
 }
 
+inline std::vector<std::string> getenv(const char *name, char split) {
```

Comment:
I don't think we should overload the function `getenv` to take a separator.  That's confusing, because these two functions do fundamentally different things.

I think a better thing to do would be to write a `split` function somewhere (probably not in this header) and let the user compose getenv and split themselves, it's not a significant amount of code.  Alternatively you could use this existing LLVM split function, https://llvm.org/doxygen/classllvm_1_1StringRef.html#a7b0138ddc186a7cb5cfd04dde671220e